presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -187,3 +187,46 @@ Accessors
 .. function:: ST_NumInteriorRing(Geometry) -> bigint
 
     Returns the cardinality of the collection of interior rings of a polygon.
+
+=====================
+Bing Tiles
+=====================
+
+A set of functions to convert between geometries and
+ `Bing tiles <https://msdn.microsoft.com/en-us/library/bb259689.aspx>`_.
+
+.. function:: bing_tile(x, y, zoom_level) => BingTile
+
+    Given XY coordinates and a zoom level, returns the corresponding Bing tile.
+    Zoom levels from 1 to 23 are supported.
+
+.. function:: bing_tile(quadKey) => BingTile
+
+    Given a QuadKey, returns the corresponding Bing tile.
+
+.. function:: bing_tile_coordinates(tile) => row(x, y)
+
+    Given a Bing tile, returns XY coordinates of that tile.
+
+.. function:: bing_tile_polygon(tile) => Geometry
+
+    Given a Bing tile, returns the polygon representation of that tile.
+
+.. function:: bing_tile_quadkey(tile) => varchar
+
+    Given a Bing tile, returns its QuadKey.
+
+.. function:: bing_tile_zoom_level(tile) => bigint
+
+    Given a Bing tile, returns its zoom level.
+
+.. function:: geometry_to_bing_tiles(geometry, zoom_level) => array(BingTile)
+
+    Given a geometry and a zoom level, returns the minimum set of Bing tiles that fully covers
+    that geometry. Zoom levels from 1 to 23 are supported.
+
+.. function:: point_to_bing_tile(point, zoom_level) => BingTile
```

Comment:
Maybe `bing_tile_at(point, zoom_level)`? BTW, is the common usage to take a `geometry` point and try to find the tile containing it or does it generally start with just the raw x and y coordinates? For the first case, taking a `point` makes sense. For the second, maybe `bing_tile_at(x, y, zoom_level)` might be more appropriate. It's also more type-safe, since it prevents anything non-point geometries being passed to the function.

Reply:
@martint , bing_tile_at(x, y, zoom_level) sounds good. I believe the common usage is to start with lat, lon pair.